configure.ac

```diff
@@ -657,7 +657,8 @@ if test "${with_bundled_protobuf}" != "no"; then
     fi
     AC_MSG_RESULT([${bundled_proto_avail}])
     if test "${with_bundled_protobuf}" == "yes" -a "${bundled_proto_avail}" != "yes"; then
-        AC_MSG_ERROR([Bundled protobuf requested using --with-bundled-protobuf but it cannot be used/found])
+        AC_MSG_WARN([Bundled protobuf requested using --with-bundled-protobuf but it cannot be used/found, skipping ACLK NG])
```

Comment:
This should still be an error. The install script _only_ passes this configure flag if the protobuf build succeeded, and if it got passed but we can’t find the files, it’s very much an error.

Reply:
I think you're right.... In the morning, I did have problem with this.... Might have been some switch I did some tries... Will look into it and revert probably. Thanks.

Reply:
Agreed with @ferroin here the flag is supposed to be 3state. If nothing given try use bundled and if not use system. This place checks case where `--with-bundled-protobuf`  was used. In such case user specifically wants to use bundled version of protobuf. In such case we need to fail with error if it is not present

Reply:
Ok, I could have sworn I had a case with it failing, without (I think) issuing `--with-bundled-protobuf`... Reverted.